en/developer_guide/multi_panel.rst

```diff
@@ -0,0 +1,104 @@
+Add panel to an administration view
+===================================
+
+Context
+-------
+
+In Open Orchestra, if you want to edit an element, you can only display the full form. In some cases
+you want to embillish the edition by splitting the full form in some smaller one.
+
+Open Orchestra already provides you a way to `use a custom Backbone view`_.
+This solution may be possible but will require a large customization of the form view.
+
+To prevent you from doing this work, Open Orchestra provides a way to add multiple tab in your edition form like in the user edit page.
+
+.. image:: ../../images/user_panel_form.png
+
+Panel System
+~~~~~~~~~~~~
+
+Each element facade contains a list of meta links that may be used to display specifics meta-informations
+or forms to add, modify or delete them.
+Most commons links are :
+
+- ``_self_add`` url to the add form action
+- ``_self_form`` url to the edit form action
+- ``_self_delete`` url to the delete form action
+- ``_self_meta`` url to meta-information
+
+All of this links are used by open orchestra javascript to send ajax call and display differents administration forms.
+Edit page of administration items use ``_self_form`` link to display the edit form.
+
+To add a new panel with your own form to this one, add a link to the facade, prefixed by ``_self_panel``.
+
+.. code-block:: php
+
+    $linkName = "_self_panel_" + $position + "_" + $panelTitle;
+    $facade->addLink($linkName, $route);
+
+The link name is a key injected in the DOM data to find the link later in javascript.
+Open Orchestra also uses this key to configure your panel:
+
+- Add a number after "_self_panel" prefixed by "_" to choose the position of your tab (begin with 1 because the ``_self_form`` link will always be in position 0)
+- Add the title of your panel after the position prefixed by "_" to choose the tab name
+
+Example
+~~~~~~~
+
+Here is an example of how the panel is used to add a workflow form to Users page.
+
+To add this link without modifying the ``UserTransformer`` which create the facade, we recommand to dispatch an event subscriber
```

Comment:
creates
recommend
an event is dispatched, a subscriber listen to events => end of the sentence to rephrase